### PR TITLE
Update expected results for functional tests

### PR DESCRIPTION
**Pull request by sschuberth**

### CODE REVIEW

File: scanner/src/main/kotlin/storages/utils/ScanResultDao.kt [hunk 1]
```diff
@@ -29,7 +29,7 @@ import org.ossreviewtoolkit.model.Identifier
 import org.ossreviewtoolkit.model.ScanResult
 
 object ScanResults : IntIdTable("scan_results") {
-    val identifier: Column<String> = text("identifier").index("identifier")
+    val identifier = text("identifier").index("identifier")
```
Comment (mnonnenmacher) [inline scanner/src/main/kotlin/storages/utils/ScanResultDao.kt:32]: Unrelated change.

Comment (mnonnenmacher) [inline scanner/src/main/kotlin/storages/utils/ScanResultDao.kt:32]: Unrelated change.

**Review by mnonnenmacher — CHANGES_REQUESTED** [review]

File: plugins/package-managers/spm/src/funTest/assets/projects/synthetic/spm-expected-output-lib.yml [hunk 2]
```diff
@@ -38,7 +38,7 @@ project:
         dependencies:
         - id: "SPM:apple:swift-atomics:1.1.0"
         - id: "SPM:apple:swift-collections:1.0.4"
-    - id: "SPM:apple:swift-nio-ssl:2.24.0"
+    - id: "SPM:apple:swift-nio-ssl:2.25.0"
```
Comment (fviernau) [inline plugins/package-managers/spm/src/funTest/assets/projects/synthetic/spm-expected-output-lib.yml:41]: Just out of curiosity: Have you been able to repro this on you machine? I saw the issue with version `2.25.0` already the day before yesterday, but locally it was still giving 2.24.0.

**Review by fviernau — APPROVED** [review]

**Review by mnonnenmacher — APPROVED** [review]

File: plugins/package-managers/spm/src/funTest/assets/projects/synthetic/spm-expected-output-lib.yml (hunk 2 above)
Comment (sschuberth) [reply plugins/package-managers/spm/src/funTest/assets/projects/synthetic/spm-expected-output-lib.yml:41]: I haven't tried (as I was on Windows) and just copied & pasted the updated result from the CI log.
